Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/DateUtils.java

```diff
@@ -0,0 +1,27 @@
+package com.mentalmachines.droidcon_boston.utils;
+
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class DateUtils {
+
+    public static Date fromISO8601UTC(String dateStr) {
+        TimeZone tz = TimeZone.getTimeZone("UTC");
+        DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.US);
+        df.setTimeZone(tz);
+
+        try {
+            return df.parse(dateStr);
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+
+        return null;
```

Comment:
A `null` value is always scary :|

Reply:
could throw the uncaught parsing exception, but added null handling if there was an invalid date in firebase